src/ts/components/subtitleoverlay.ts

```diff
@@ -516,19 +528,80 @@ class SubtitleRegionContainer extends Container<ContainerConfig> {
     }, this.config);
   }
 
-  addLabel(labelToAdd: SubtitleLabel) {
+  private addCueBoxContainer(labelToAdd: SubtitleLabel) {
+    const id = labelToAdd.getConfig().id;
+    this.cueBoxContainers[id] = new SubtitleCueBoxContainer();
+    this.cueBoxContainers[id].addLabel(labelToAdd);
+    setVttCueBoxStyles(this.cueBoxContainers[id], labelToAdd.vtt);
+    this.addComponent(this.cueBoxContainers[id]);
+    this.updateComponents();
+
+    if (labelToAdd.vtt.region && labelToAdd.vtt.region.scroll) {
```

Comment:
What does scroll mean exactly? (There is no documentation in the player)

Reply:
Means that the bottom text pushes the top one (in our css means that we scroll the subtitle container to the bottom) and hide the subtitles on the top